deepchecks/nlp/metric_utils/token_classification.py

```diff
@@ -21,12 +21,7 @@
 __all__ = ['get_default_token_scorers', 'validate_scorers', 'get_scorer_dict']
 
 DEFAULT_AVG_SCORER_NAMES = ('f1_macro', 'recall_macro', 'precision_macro')
-DEFAULT_PER_CLASS_SCORER_NAMES = tuple()
-
-# see issue DEE-473
-# https://linear.app/deepchecks/issue/DEE-473/incorrectly-inferred-model-classes-for-token-classification-task
-#
-# DEFAULT_PER_CLASS_SCORER_NAMES = ('f1_per_class',)
+DEFAULT_PER_CLASS_SCORER_NAMES = ('f1_per_class', 'precision_per_class', 'recall_per_class')
```

Comment:
shouldn't they be token_f1_per_class and such? @noamzbr

Reply:
Yep think Nadav is correct, original value was wrong also ):

Reply:
Mmm I see @yromanyshyn added `return [f'token_{it}' for it in names]` in line 105 which makes it work. I know it was me that decided to call these scorers "token_*" but I'm not sure why it's necessary. @yromanyshyn any reason why not to just call them with a regular name?

Reply:
@noamzbr 

>@yromanyshyn any reason why not to just call them with a regular name?

hm, no do not see any reason why should they start with `token`
I will rename

Reply:
renamed